src/index.js

```diff
@@ -28,9 +28,13 @@ export default class Fetch {
     return new Fetch(defaultURL, defaultHeaders, options)
   }
 
-  buidAuthorizationHeader(key) {
+  buildAuthorizationHeader(key) {
     if (key) {
-      const authKeyword = this.options.defaultAuthorizationKeyword || DEFAULT_AUTHORIZATION_KEYWORD
+      const authKeyword = (
```

Comment:
I think it is possible, but no problem, if we've got this problem, we will fix it.